src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -102,8 +107,16 @@ function dismissError(item) {
 
     if (item.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
         PaymentMethods.clearDeletePaymentMethodError(paymentList, paymentID);
+        // as part of migrating cardList to fundList key, we will duplicate the action with both keys temporarily.
+        if (!isBankAccount) {
+            PaymentMethods.clearDeletePaymentMethodError(ONYXKEYS.FUND_LIST, paymentID);
+        }
     } else {
         PaymentMethods.clearAddPaymentMethodError(paymentList, paymentID);
+        // as part of migrating cardList to fundList key, we will duplicate the action with both keys temporarily.
+        if (!isBankAccount) {
+            PaymentMethods.clearAddPaymentMethodError(ONYXKEYS.FUND_LIST, paymentID);
+        }
```

Comment:
**recommendation:** create a follow up issue and in the description list out a checklist with the various places we will need to return and fix up comments/code like this.